.github/workflows/release.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
       - name: Install Pulumi CLI
         uses: pulumi/action-install-pulumi-cli@b374ceb6168550de27c6eba92e01c1a774040e11 # tag=v2.0.0
       - name: Setup Node
-        uses: actions/setup-node@2fddd8803e2f5c9604345a0b591c3020ee971a93 # tag=v3.4.1
+        uses: actions/setup-node@969bd2663942d722d85b6a8626225850c2f7be4b # tag=v3.4.1
```

Comment:
@dirien I would have expected that Dependabot would update the tag name in the comment to remain in sync. Wrong expectations on my side?

Reply:
I mixed it up with the renovate bot!

There is a Issue https://github.com/dependabot/dependabot-core/issues/4691 regarding this, and I plan on creating a PR for this. I think this is very important.

So for now pining to commit hash is for me mandatory best practice in terms of supply chain security but currently hard to challenging to read.

Again, my PR will bring the renovate feature to dependabot too!